src/model.ts

```diff
@@ -177,7 +180,8 @@ export function convertDescribeJobtoJobDetail(
     updateTime: describeJob.update_time,
     startTime: describeJob.start_time,
     endTime: describeJob.end_time,
-    scheduleInterval: 'weekday'
+    scheduleInterval: 'weekday',
+    downloaded: describeJob.downloaded
```

Comment:
`downloaded` could also become optional; `undefined` and `false` would both mean "not downloaded." `undefined` would also imply "not applicable," such as for a job that's still running.

Reply:
downloaded will always be present, default is false.